doc/man5/config.pod

```diff
@@ -542,22 +443,20 @@ Ignored in set-user-ID and set-group-ID programs.
 
 =item B<OPENSSL_CONF_INCLUDE>
 
-The optional path to prepend to all .include paths.
+The optional path to prepend to all B<.include> paths.
 
 =back
 
 =head1 BUGS
 
-Currently there is no way to include characters using the octal B<\nnn>
-form. Strings are all null terminated so nulls cannot form part of
-the value.
+There is no way to include characters using the octal B<\nnn> form. Strings
+are all null terminated so nulls cannot form part of the value.
 
 The escaping isn't quite right: if you want to use sequences like B<\n>
 you can't use any quote escaping on the same line.
 
-Files are loaded in a single pass. This means that an variable expansion
-will only work if the variables referenced are defined earlier in the
-file.
+Files are loaded in a single pass. This means that variable expansion
```

Comment:
I would say this does not belong to BUGS section as this is rather a quite usual and expected feature of such config file loaders.